tokio/src/runtime/task/join.rs

```diff
@@ -182,28 +182,29 @@ impl<T> JoinHandle<T> {
     /// ```rust
     /// use tokio::time;
     ///
-    /// #[tokio::main]
-    /// async fn main() {
-    ///    let mut handles = Vec::new();
-    ///
-    ///    handles.push(tokio::spawn(async {
-    ///       time::sleep(time::Duration::from_secs(10)).await;
-    ///       true
-    ///    }));
-    ///
-    ///    handles.push(tokio::spawn(async {
-    ///       time::sleep(time::Duration::from_secs(10)).await;
-    ///       false
-    ///    }));
-    ///
-    ///    for handle in &handles {
-    ///        handle.abort();
-    ///    }
-    ///
-    ///    for handle in handles {
-    ///        assert!(handle.await.unwrap_err().is_cancelled());
-    ///    }
+    /// # #[tokio::main(flavor = "current_thread")]
+    /// # async fn main() {
+    /// # tokio::time::pause();
```

Comment:
Now that we're hiding the main macro, we can do this instead:
```suggestion
    /// # #[tokio::main(flavor = "current_thread", start_paused = true)]
    /// # async fn main() {
```